public/game/actions.js

```diff
@@ -393,6 +408,16 @@ function reshuffleAndDraw(state) {
 	return drawCards(nextState)
 }
 
+function endEncounter(state) {
+	const nextState = produce(state, (draft) => {
+		draft.hand = []
+		draft.discardPile = []
+		draft.ExhaustPile = []
+		draft.drawPile = shuffle(draft.deck)
+	})
+	return drawCards(nextState)
+}
+
```

Comment:
This method effectively replaces `reshuffleAndDraw`. How about we just add `draft.ExhaustPile = []` to the existing method? Since that's already part of the tests etc.